image/manifest.go

```diff
@@ -89,7 +89,7 @@ func (m *manifest) validate(w walker) error {
 
 func (m *manifest) unpack(w walker, dest string) error {
 	for _, d := range m.Layers {
-		if d.MediaType != string(schema.MediaTypeImageConfig) {
+		if d.MediaType != string(schema.MediaTypeImageLayer) {
 			continue
```

Comment:
@wking   If the spec does not limit the allowed types, I don't think we should error out on unpacking, we should just
unpacking the allowed typed layers.  But I think this is another issue we should discuss more not related to this pr